### ISSUE TITLE
geojsonsf

### ISSUE THREAD

**Issue by SymbolixAU** [ateucher/rmapshaper#81]
In the readme you identify the conversion to/from GeoJSON is slow

>  They are slower with the Spatial classes due to internal conversion to/from json

Other than the [date fix](https://github.com/SymbolixAU/geojsonsf/pull/33) (thanks again!), is there anything you would like added to [geojsonsf](https://github.com/SymbolixAU/geojsonsf) so you can use it in `rmapshaper` to do the GeoJSON <--> sf conversion?

**Comment by ateucher** [comment]
Working in `geojsonsf` branch. 

To Do:

- [ ] remove support for geojson lists
- [ ] drop geojsonio (possibly to `Suggests` for now as it's useful in tests)
- [ ] Wait for https://github.com/SymbolixAU/geojsonsf/issues/79 to be fixed

**Comment by dcooley** [comment]
Would you like me to add a `json` class` to geojsonsf ?

**Comment by ateucher** [comment]
Sorry @dcooley I was wrong - `geojsonsf` already adds `json` class:

``` r
library(sf)
#> Linking to GEOS 3.8.1, GDAL 2.4.4, PROJ 7.0.0
library(geojsonsf)

xs <- st_polygon(list(cbind(approx(c(0, 0, 1, 1, 0))$y,
                            approx(c(0, 1, 1, 0, 0))$y)))
xsf <- st_sf(geometry = st_sfc(xs, xs + 2, xs + 3), a = 1:3)

class(geojsonsf::sf_geojson(xsf))
#> [1] "geojson" "json"
```

<sup>Created on 2020-04-17 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by ateucher** [comment]
Checklist updated - rmapshaper can just have methods for `geojson` I think...

**Comment by ateucher** [comment]
This will be closed by #118
